### ISSUE TITLE
Column inconsistencies in current `histologies-base-adapt.tsv`

### ISSUE THREAD

**Issue by jharenza** [PediatricOpenTargets/ticket-tracker#330]
#### What data file(s) does this issue pertain to?
`histologies-base-adapt.tsv`

#### What release are you using?

pull from 2022-05-23

#### Put your question or report your issue here.
`cns_region` should be `CNS_region` based on past files, but since we are adding this after pulling the base file, maybe we should just remove this column from the data warehouse table?
`parental_aliquot_id` is missing, but this may be expected?
`broad_composition` is no longer used, so we can remove from the table

**Comment by nicholasvk** [comment]
cns_region - Yes, looks like this changed when I needed to create a staging table with column names that conform to database standards (no capitalization). I can remove it, but this means we need to remove it for all all cohorts in openpedcan and just want to confirm that is OK.

parental_aliquot_id : This was a historical field we removed a couple of years ago. It was populated with SDG ID because at the time many sample_ids and aliquot_ids had additional info appended. We worked to remove the appended info and the stand alone parent aliquot id field was no longer needed.

broad_composition - Yes, this is just a placeholder field. Like cns_region, I would just want to confirm that its OK to remove from all openpedcan cohorts. My understanding though was it was helpful to have these as placeholders fields for downstream analysis?
